jwst/outlier_detection/_fileio.py

```diff
@@ -20,13 +14,53 @@ def save_median(median_model, make_output_path, asn_id=None):
     median_model : ~jwst.datamodels.ImageModel
         The median ImageModel or CubeModel to save
     '''
-    default_suffix = "_outlier_i2d.fits"
-    if asn_id is None:
-        suffix_to_remove = default_suffix
+    median_model = ImageModel(median_data)
+    median_model.update(example_model)
+    median_model.meta.wcs = median_wcs
+    output_path = intermediate_output_path(example_model, "median", make_output_path)
```

Comment:
Similarly this output path is `jw03516001001_02101_00003_nrca1_a3001_a3001_a3001_median.fits` (note the 3 association ids).

Reply:
resolving because this issue is being tracked by the comments above